frappe/public/js/frappe/ui/keyboard.js

```diff
@@ -160,6 +160,20 @@ frappe.ui.keys.add_shortcut('alt+h', function(e) {
 	$('.dropdown-help a').eq(0).click();
 }, __('Help'));
 
+frappe.ui.keys.on('shift+>',()=> {
```

Comment:
Move these shortcuts to `form.js` and use `frappe.ui.keys.add_shortcut`

Reply:
Done